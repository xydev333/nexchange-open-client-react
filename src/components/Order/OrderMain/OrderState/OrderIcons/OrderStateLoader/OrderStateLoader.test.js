import React from 'react';
import { shallow } from 'enzyme';
import OrderStateLoader from './OrderStateLoader';

describe('OrderStateLoader', () => {
  it('renders correctly', () => {
    expect(shallow(<OrderStateLoader />)).toMatchSnapshot();
  });
});
