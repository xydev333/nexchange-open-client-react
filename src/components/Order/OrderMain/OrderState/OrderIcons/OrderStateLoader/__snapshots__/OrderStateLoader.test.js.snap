// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderStateLoader renders correctly 1`] = `ShallowWrapper {}`;
